terraform/environments/core-logging/logging.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   resolver_query_log_config_names = toset(["core-logging-rlq-cloudwatch", "core-logging-rlq-s3"])
-  vpc_ids                         = { for key, value in module.vpc : key => value["vpc_id"] if key == "live_data" }
```

Comment:
This feels superfluous; the `if` statement is meant to only extract the `live_data` VPC. You can remove it entirely if you want to capture both VPC IDs in the local value.